theme.json

```diff
@@ -68,6 +68,21 @@
 				}
 			]
 		},
+		"custom": {
+			"typography": {
+				"font-weight": {
```

Comment:
I actually don't think there's a lot of value added by separating font weight within custom, as these values are "set in stone". For example, extraLight will always have a font-weight of 200. I'd rather us use the actual values throughout instead.

Reply:
This system sets us up so that style variations could potentially set their own `custom.typography.font-weight.regular` to `430` if they desire. 🤔

Reply:
I know this because Outfit is generally a thicker font, so I tried setting custom weights in theme.json to `150`, `250`, `350`, `450`, etc and those were ignored when I selected weights via the Appearance in editor. Medium was hardcoded inline as `500`, even though I had it at `450` in theme.json.

Reply:
Yes, I can see how the current implementation in Gutenberg is limiting and only allows the UI to show and set traditional values.